app/controllers/posts_controller.rb

```diff
@@ -1,6 +1,6 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
+    @posts = Post.all.includes(:comments)
```

Comment:
- Please use the User object to fetch all posts for that user as suggested earlier.
```ruby
@user = User.find(params[:user_id])
@posts = @user.posts.includes(:comments)
```

Reply:
@whiz25 
thanks for your feedback.
done